ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -258,6 +261,51 @@ function start_osd {
 }
 
 
+########################
+# OSD_DIRECTORY_SIMPLE #
+########################
+
+function osd_directory_simple {
+  if [[ ! -d /var/lib/ceph/osd ]]; then
```

Comment:
Perhaps:
`$(find /var/lib/ceph/osd -type d |wc -l) -gt 1`

Basically, check that the directory `/var/lib/ceph/osd` contains at least one subdirectory.